routes/web.php

```diff
@@ -20,3 +20,4 @@
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/track', [App\Http\Controllers\HomeController::class, 'index'])->name('track');
```

Comment:
@hayato26 
Controllerの指定の仕方が違いますね！
これはLaravelの基礎中の基礎なのでControllerの動画見返しておいてください！


```suggestion
Route::get('/track', [App\Http\Controllers\TrackController::class, 'index'])->name('track');
```

Reply:
@issiyda 
ありがとうございます！
表示できました！